tests/publiccloud/patch_and_reboot.pm

```diff
@@ -32,6 +32,9 @@ sub run {
 
     ssh_fully_patch_system($remote);
 
+    record_info('UNAME', $args->{my_instance}->ssh_script_output(cmd => 'sudo uname -a'));
+    record_info('RPM', $args->{my_instance}->ssh_script_output(cmd => 'sudo rpm -qa'));
```

Comment:
This is a looooong output, on my laptopt it's almost 7000 lines.

If necessary, I'd upload it as a log file.

Reply:
+1

Reply:
Please re-check the code.